frontend/public/components/edit-yaml.jsx

```diff
@@ -685,6 +732,11 @@ export const EditYAML = connect(stateToProps)(
               <div className="co-p-has-sidebar">
                 <div className="co-p-has-sidebar__body">
                   <div className="yaml-editor" ref={(r) => (this.editor = r)}>
+                    <div className="yaml-editor__links">
+                      {shortcuts}
+                      {showSidebar !== true && hasSidebarContent && divider}
+                      {!showSidebar && hasSidebarContent && sidebarLink}
```

Comment:
I don't understand why the check for the divider and the sidebar link is different. Can't this be one check for both?